README.rst

```diff
@@ -5,6 +5,25 @@ A sunpy-affiliated package for solar instrument-specific tools.
     :target: http://www.sunpy.org
     :alt: Powered by SunPy Badge
 
+What is sunkit-instruments?
+---------------------------
+
+sunkit-instruments is a sunpy-affiliated package for solar instrument-specific tools.
+Its purpose is not to be a repository for all tools for all instruments.
+Instead it is intended to perform three main roles:
+
+1. Hold instrument tools that are so few they do not justify their own package;
```

Comment:
I think I agree with @nabobalis  -' justify' might be a wrong word here, maybe something like 'require their own package'?